src/meta/src/barrier/recovery.rs

```diff
@@ -64,8 +64,12 @@ where
         debug!("recovery start!");
         let retry_strategy = Self::get_retry_strategy();
         let (new_epoch, responses) = tokio_retry::Retry::spawn(retry_strategy, || async {
-            let info = self.resolve_actor_info(None).await;
```

Comment:
And I suggest to add an option named `enable_auto_migration_for_recovery` for `BarrierManager` to control whether we should do auto migration.